app/perfaddresscache.h

```diff
@@ -42,17 +42,22 @@ class PerfAddressCache
 
     struct SymbolCacheEntry
     {
-        SymbolCacheEntry(quint64 offset = 0, quint64 size = 0, const QByteArray &symname = {})
+        SymbolCacheEntry(quint64 offset = 0, quint64 value = 0, quint64 size = 0, const QByteArray &symname = {}, const QByteArray &mangledSymName = {})
             : offset(offset)
+            , value(value)
             , size(size)
             , symname(symname)
+            , mangledSymName(mangledSymName)
```

Comment:
this is still including the mangled sym name - can we please first implement the demangler without that? As I said - in theory start address and end address is all we need for the disassembler. If more is needed for some corner cases, then we can think about adding that in the future, but it would at the very least require a proper explanation (which I still haven't seen).